src/containers/post/post.component.jsx

```diff
@@ -46,6 +50,23 @@ const Post = ({ post, user, fetchLatestPosts }) => {
     fetchAvatar();
   }, []);
 
+  useEffect(() => {
+    if (img_post) {
+      storage
+        .ref(`posts/${picID}-01`)
+        .getDownloadURL()
+        .then((img1Url) => {
+          setImg1(img1Url);
+        });
+      storage
+        .ref(`posts/${picID}-02`)
+        .getDownloadURL()
+        .then((img2Url) => {
+          setImg2(img2Url);
+        });
```

Comment:
This block of code should be within the `then` block of the previous part. Or you can use the async await syntax to make it all cleaner